script/Arcueid/Day 4/04_00_ARC04_1B.txt

```diff
@@ -226,27 +241,33 @@ Looks like Kohaku-san is seriously angry.
 [sha:9d9fa38b39f1023846fc5efc2fd584bd7f16480f]{
 -- Page 44, Offset 5674.
 -- ……そういえば風呂場で見た自分の顔は、青を通り越して土気色っぽかったっけ……。
-...I remember when I saw my own face in the bathroom, it was beyond blue, a deathly pale color...
+...Thinking back on it,
+ when I saw my face in the bathroom it wasn't just pale, it was positively ashen...
 }
 [sha:b567fadce8bda9309b5d2a58bfbdfd5abd036fc3]{
 -- Page 45, Offset 5675.
 -- 「いや、でも大丈夫だって。俺はもともと血の気が少ないし、人より顔色が悪く見えるだけというか、」
-"But it's okay. I've always had a shortage of blood so I look paler than most people."
+"Come on, it's not that bad.
+ My complexion has always been a little on the pale side,
+ so I might look a little off-color compared to some people but―――"
 }
 [sha:fd99179705999e72978f204cc79b06d16221bc68]{
 -- Page 46, Offset 5676.
 -- 「だめです！　朝ごはんを食べないと大きくなれないんですから！　食欲がないようでしたら病人食にいたしますから、どうぞ食堂にいらしてください」
-"No! You won't grow up without eating breakfast! If you don't have much of an appetite, I can make you a light meal, so please come to the dining room."
+"No!
+ You won't grow if you don't eat your breakfast!
+ If you don't have much of an appetite I'll prepare you something easy to digest,
+ so please make your way to the dining room."
 }
 [sha:7847778b26fd6e9664c069ad458bf1610794a61f]{
 -- Page 47, Offset 5677.
 -- 琥珀さんは俺の腕を掴んだまま、強引に食堂へ歩きだした。
-Kohaku-san grabs my arm and forcefully starts walking to the dining room.
+Keeping a firm grip on my arm, Kohaku-san starts frogmarching me toward the dining room.
```

Comment:
to the dining room

Reply:
Hmm for some reason `to` just doesn't feel _right_ here to me. 3rd opinion?